Description: Include MPI headers when building Oct/MEX-files
Author: Thomas Weber <tweber@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=598227
Forwarded: not-needed
Last-Update: 2012-08-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/mkoctfile.in
+++ b/src/mkoctfile.in
@@ -77,7 +77,7 @@
 : ${CXXFLAGS=%OCTAVE_CONF_CXXFLAGS%}
 : ${CXXPICFLAG=%OCTAVE_CONF_CXXPICFLAG%}
 : ${XTRA_CFLAGS=%OCTAVE_CONF_XTRA_CFLAGS%}
-: ${XTRA_CXXFLAGS=%OCTAVE_CONF_XTRA_CXXFLAGS%}
+: ${XTRA_CXXFLAGS=-I/usr/include/mpi %OCTAVE_CONF_XTRA_CXXFLAGS%}
 
 : ${DEPEND_FLAGS=%OCTAVE_CONF_DEPEND_FLAGS%}
 : ${DEPEND_EXTRA_SED_PATTERN=%OCTAVE_CONF_DEPEND_EXTRA_SED_PATTERN%}
